apps/admin/frontend/src/components/reporting/pdf_viewer.tsx

```diff
@@ -0,0 +1,127 @@
+import { Document, Page, pdfjs } from 'react-pdf';
+import styled from 'styled-components';
+import React, { useState } from 'react';
+import { Button } from '@votingworks/ui';
+import { PreviewLoading } from './shared';
+
+pdfjs.GlobalWorkerOptions.workerSrc = '/pdfjs-dist/pdf.worker.js';
+
+const ThumbnailContainer = styled.div`
+  margin: 1rem 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+  .thumbnail {
+    box-shadow: 0 3px 10px rgb(0, 0, 0, 20%);
+  }
+`;
+
+const PageNumberControl = styled.div`
+  position: fixed;
+  right: 0.5rem;
+  bottom: 0.5rem;
+  height: 2.5rem;
+  border-radius: 0.5rem;
+  background: white;
+  box-shadow: 0 3px 10px rgb(0, 0, 0, 20%);
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: stretch;
+
+  .page-control-button {
+    width: 2rem;
+    padding: 0;
+
+    :disabled {
+      border: none;
+    }
+  }
+`;
+
+const PageCount = styled.div`
+  display: flex;
+  align-items: center;
+  line-height: 1.5rem;
+  margin: 0 0.5rem;
+`;
+
+export interface PdfViewerProps {
+  pdfResourcePath?: string;
+}
+
+export function PdfViewer({ pdfResourcePath }: PdfViewerProps): JSX.Element {
+  const [numPages, setNumPages] = useState(1);
+  const [pageNumber, setPageNumber] = useState(1);
+
+  function canPageBack() {
+    return pageNumber > 1;
+  }
+
+  function pageBack() {
+    if (canPageBack()) {
+      setPageNumber(pageNumber - 1);
+    }
+  }
+
+  function canPageForward() {
+    return pageNumber < numPages;
+  }
+
+  function pageForward() {
+    if (pageNumber < numPages) {
+      setPageNumber(pageNumber + 1);
+    }
+  }
+
+  if (!pdfResourcePath) {
+    return <PreviewLoading />;
+  }
+
+  return (
+    <React.Fragment>
+      <Document
+        file={`${window.location.protocol}//${window.location.host}/${pdfResourcePath}`}
+        loading={<PreviewLoading />}
+        onLoadSuccess={(page) => setNumPages(page.numPages)}
+      >
+        <ThumbnailContainer>
+          <Page
+            pageNumber={pageNumber}
+            scale={2.3}
+            renderMode="svg"
```

Comment:
SVG render mode is deprecated, but the default `canvas` mode makes the fonts look grainy. Interestingly, this is only in `kiosk-browser`, not Chrome. So maybe there's some difference in `canvas` font handling across Chrome versions? Anyway, using SVG mode generates a bunch of console warnings but it works great. I've troubleshot the `canvas` mode by playing with dimensions and scaling, but couldn't get it to look clean.